### PR TITLE
Edit User Profiles

### PR DESCRIPTION
**Pull request by drg226**
editing the clean way.

### CODE REVIEW

File: .idea/knowhere.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (KingStrafeIV) [inline .idea/knowhere.iml]: This is a preference file and shouldn't be committed

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bzumhagen) [inline .idea/misc.xml:1]: This is a preference file and shouldn't be committed

File: .idea/knowhere.iml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bzumhagen) [inline .idea/knowhere.iml:1]: This is a preference file and shouldn't be committed

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bzumhagen) [inline .idea/modules.xml:1]: This is a preference file and shouldn't be committed

File: .idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bzumhagen) [inline .idea/vcs.xml:1]: This is a preference file and shouldn't be committed

File: .idea/workspace.xml [hunk 6]
```diff
@@ -0,0 +1,1032 @@
...
+    <entry file="file://$PROJECT_DIR$/website/migrations/0012_auto_20151116_2046.py">
+      <provider selected="true" editor-type-id="text-editor">
+        <state vertical-scroll-proportion="0.31390134">
+          <caret line="16" column="10" selection-start-line="16" selection-start-column="10" selection-end-line="16" selection-end-column="10" />
+          <folding />
+        </state>
+      </provider>
+    </entry>
+  </component>
+</project>
```
Comment (bzumhagen) [inline .idea/workspace.xml:1032]: This is a preference file and shouldn't be committed

File: knowhere/settings.py [hunk 7]
```diff
@@ -114,7 +114,7 @@
         'ENGINE': 'django.db.backends.mysql',
         'NAME': 'knowhere',
         'USER': 'knowhere_system',
-        'PASSWORD': '!catjetpacks123',
+        'PASSWORD': '!catjets1234',
```
Comment (bzumhagen) [inline knowhere/settings.py:117]: Why change the password?

Comment (drg226) [inline knowhere/settings.py:117]: Yeah these slipped through in the commit. I'm not sure how my db got saved
with a different password but I've just been lazy about changing it.

File: website/models.py [hunk 8]
```diff
@@ -103,4 +103,8 @@ class NotificationZone(models.Model):
     radius = models.DecimalField(max_digits=9, decimal_places=2)
 
     def __str__(self):
-        return self.name
\ No newline at end of file
+        return self.name
+
+class User(models.Model):
```
Comment (bzumhagen) [inline website/models.py:108]: Why are you creating a new user model? We are currently using the default django user model for everything so far.

File: website/forms.py [hunk 9]
```diff
@@ -1,11 +1,11 @@
-from website.models import NotificationZone, Group, Notification
+from website.models import NotificationZone, Group, Notification, User
```
Comment (bzumhagen) [inline website/forms.py:1]: Should be importing from django not the Models.

**Comment by drg226** [comment]
closing.

**Pull request closed without merging**